crml/nft/src/tests.rs

```diff
@@ -177,6 +177,7 @@ fn create_collection_royalties_invalid() {
 	ExtBuilder::default().build().execute_with(|| {
 		let owner = 1_u64;
 
+		//Too big royalties should fail
```

Comment:
nit: space after the `//`
```suggestion
		// Too big royalties should fail
```